socorro/tests/processor/test_processor_app.py

```diff
@@ -99,15 +101,20 @@ def test_transform_success(self):
         pa.processor.process_crash = mocked_process_crash
         pa.destination.save_processed_crash = mock.Mock()
         finished_func = mock.Mock()
-        patch_path = "socorro.processor.processor_app.os.unlink"
```

Comment:
The processor app is no longer in charge of deleting files manually from the dumps values. Instead, it does it automatically when the temporary directory context exits.

Given that, we don't need to add this testing.